data/zilliqa/assets/zil133vrh59edllqz7guq7htfthjmj7gct5jxqqtl7

```diff
@@ -0,0 +1 @@
+Flat token
```

Comment:
dafuk is this? remove

Reply:
Hi Balthazar, apologies for all of my mess-ups there. I really didn't mean to spam you there - I am just rather unfamiliar with github and have been trying to copy basically what every other token has done but for somereason I didnt have the upload button to submit by JSON & logo files. However, I think I finally found it now.